README.md

```diff
@@ -1,3 +1,10 @@
-# Normal Depth Map
+Foward looking sonar for gazebo
+=======================================
 
-This repository contains the code for normal depth map on OSG scene handled by shaders.
+Sonar based on the article **[Custom Shader and 3D Rendering for computationally efficient Sonar Simulation](http://sibgrapi.sid.inpe.br/col/sid.inpe.br/sibgrapi/2016/08.09.11.51/doc/sibgrapi16.pdf)**.
```

Comment:
In fact, the sonar is based on

A novel GPU-based sonar simulator for real-time applications 
R. Cerqueira, T. Trocoli, G. Neves, S. Joyeux, J. Albiez and L. Oliveira 
Elsevier Computers & Graphics Journal 
Special Section on SIBGRAPI 2017

Reply:
But this one is open for everyone?

Reply:
It is open once you have access to ScienceDirect (like IEEE Xplore).

Reply:
Do you see a problem if I cite both? For those who don`t have access to ScienceDirect

Reply:
I suggest to cite the last paper, because the first one does not cover all current development...

Reply:
Feel free to decide, it is just a suggestion.